data-structures-stacks/is-empty.js

```diff
@@ -0,0 +1,7 @@
+export default function isEmpty(stack) {
+  if (stack.peek() === undefined) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
Don't use `if (x) return true; else return false;`. It's ugly. Just `return x;`. If you must return a boolean and `x` is not a boolean, use `return !!x;`.